### ISSUE TITLE
8.12.4 does not start after update neither on win 7x32 nor on win10x64

### ISSUE THREAD

**Issue by m3dbedb** [a85/POSTMan-Chrome-Extension#10224]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

Postman have updated automatically in july. In july 2021 I have previous version worked, but now I see only blank black box where usually was shown postman starting message.

### Steps To Reproduce

1.Install postman 8.10.0 for windows
2.start it
3.see error

### Environment Information

```markdown
- Operating System: win 7 32, win 10 64
- Platform Type: physical machines based on core i3 and core i5 with 3 - 8 GB RAM
- Postman Version: 8.10.0
```

### Additional Context?

Deinstalling and installing again does not help, the only way to work was deinstaling 8.10.0 and installing 8.6.0

**Comment by postman-staff** [comment]
Hi  @m3dbedb,

We have identified that such issues occur with specific GPUs and usually disabling GPU rendering for Postman fixes the problem.

Would you be able to try and follow the steps in this article- https://support.postman.com/hc/en-us/articles/360026088854-Postman-is-stuck-on-the-load-screen and see if disabling the GPU rendering fixes the problem?

Thanks! Please let me know how it goes.

via @ash007-postman | Postman Support

**Comment by m3dbedb** [comment]
As you mention, I have to start postman 8.10 to open setting and disable hardware acceleration (HA).
I can't neither open postman 8.10 nor disable HA in intel intergated graphics, there is no such option. (I have intel HD graphics 2000, driver version 9.17.10.4229)

**Comment by postman-staff** [comment]
Hi  @m3dbedb,

You will have to add an environment variable `POSTMAN_DISABLE_GPU=true`Please find below the steps to do the same:

### 1. Open advanced system settings
 ![](https://user-images.githubusercontent.com/5207331/40467461-aad4b7ec-5f47-11e8-9382-54383f4ca433.png)

### 2. In the `Advanced` tab click `Environment Variables`
 ![](https://user-images.githubusercontent.com/5207331/40467512-db8cb114-5f47-11e8-9138-c043f7965771.png)

### 3. Add a new variable
 ![](https://user-images.githubusercontent.com/5207331/40467641-50ecb210-5f48-11e8-97ef-4a2728f36183.png)

### 4. Logout and Login back or Restart your PC


Regards,
Asha Singh

Status Page | Learning Center | YouTube | Twitch | Bugs + Feature Requests | Community

* * *

Product Support Business Hours: `Monday, 3:30 AM UTC - Friday, 11:30 PM UTC`

**Comment by Rory-Sullivan** [comment]
I had the same issue and turning off GPU acceleration worked for me, thank you.

P.S. I am running Postman on an AWS Windows Workspace

**Comment by postman-staff** [comment]
Hi @m3dbedb,

We haven’t heard back from you, I hope that you were able to get this sorted by following the provided steps.

Do let us know.

via @ash007-postman | Postman Support

**Comment by postman-staff** [comment]
Hi @m3dbedb,

I'll be closing this conversation from our end as we haven't heard back. If the issue persists, please reach out to us by submitting a ticket here: https://www.postman.com/support ​so that we can investigate further.

Have a nice day.

via @ash007-postman | Postman Support

**Comment by m3dbedb** [comment]
the issue is still actual. Release notes states actual postman version is 9.0.5, but for win32 available only 8.12.4 and it does not start, hangs on 

Preparing your workspaces...
This might take a few minutes.

gpu disabled, system reboot does not help

Currently I have to disable auto update (because postman updates without permission an breakes again) and get back to 8.6.0 as long as it it the latest working version for me.
